packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx

```diff
@@ -62,14 +75,32 @@ export const EmbeddedPeoplePane = (props: {
         onPeopleButtonClicked={props.onPeopleButtonClick}
       >
         <Stack verticalFill tokens={peoplePaneContainerTokens}>
-          {participantList}
+          <Stack.Item>{participantList}</Stack.Item>
+          {
+            // Only render LocalAndRemotePIP when this component is NOT hidden because VideoGallery needs to have
+            // possession of the dominant remote participant video stream
+            !props.hidden && (
+              <Stack horizontalAlign="end" styles={localAndRemotePIPContainerStyles}>
+                <Stack styles={localAndRemotePIPStyles}>
+                  <LocalAndRemotePIP
+                    {...pictureInPictureProps}
+                    {...pictureInPictureHandlers}
+                    // eslint-disable-next-line @typescript-eslint/no-empty-function
+                    onClick={() => {}} // no onClick behavior
+                  />
+                </Stack>
+              </Stack>
+            )
+          }
           {inviteLink && (
-            <PrimaryButton
-              onClick={() => copy(inviteLink)}
-              styles={copyLinkButtonStyles}
-              onRenderIcon={() => <FontIcon iconName="Link" style={linkIconStyles} />}
-              text="Copy invite link"
-            />
+            <Stack.Item styles={copyLinkButtonContainerStyles}>
+              <PrimaryButton
+                onClick={() => copy(inviteLink)}
+                styles={copyLinkButtonStyles}
+                onRenderIcon={() => <FontIcon iconName="Link" style={linkIconStyles} />}
+                text="Copy invite link"
```

Comment:
Not for this PR: This string needs to be localized.

Reply:
It should be localized already somewhere as call composite uses this (but yes in a separate PR would be better)

Reply:
This string is localized in PR #1515